WGS_pipeline/mutation_pipeline.md

```diff
@@ -0,0 +1,49 @@
+# mutation pipeline eplaination:
+
+## comparison of mutect2 with the previous pipeline:
+
+Here is a rough comparison of their output as a slide deck: 
+[DepMap CGA Mutect2 Comparision](https://docs.google.com/presentation/d/1_dS3HBmq9XwdC7iRvhXV5lj6RAET57hainiEWweHaEQ/edit#slide=id.g11968108cb5_0_0)
+
+
+## diagram of the pipeline
+
+![](../data/CCLE3_mut.png)
+
+
+## comparison of the MAF output to the previous dataset's MAF (pre 22Q4)
+
+1. dropped: "NCBI_Build": the same everywhere
+2. dropped "End_Position", "Entrez_Gene_Id" (not useful)
+3. dropped "Variant_Type" as it is not straight forward anymore an not given by annotators (harder to infer)
+4. dropped "PASS"
+5. dropped: "TCGAhsCnt", "isTCGAhotspot" (the same as the cosmic version and not available as default from most annotators)
+6. dropped "Codon_Change", "cDNA_Change" as same as DNAchange
+7. added many new columns: see [CCLE MAF colunms selection](https://docs.google.com/spreadsheets/d/1qP7-SWlUrSH85KWzK1aTRb_N6yvvE6yB6JfN-JUekt8/edit#gid=1503712741)
+8. renamed columns: ‘dbSNP_RS’,‘dbSNP_Val_Status’,
```

Comment:
what are they renamed into?

Reply:
It is all out of date so better use the DMC 22Q4 document and copy some stuff here I think.